sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -797,7 +795,7 @@
           {
             "name": "If-Match",
             "in": "header",
-            "description": "The Etag to be used to determine if the replace operation should be carried out",
+            "description": "A string representing a weak ETag for the device twin, as per RFC7232. It determines if the replace operation should be carried out.",
```

Comment:
Follow this format - A string representing a weak ETag for xxxx, as per RFC7232. Should not be set when creating xxxx, but may be set when updating a xxxx

This has been documented to keep consistency - [Common Swagger Descriptions](https://microsoft.sharepoint.com/teams/IotHub22/_layouts/OneNote.aspx?id=%2Fteams%2FIotHub22%2FSiteAssets%2FIot%20Hub%20Notebook&wd=target%28PnP.one%7CFF56CB75-EE56-4A6F-AE61-85C3E2CE07A5%2FCommon%20swagger%20descriptions%7C9D7DB953-F12B-4BD6-B7EC-59619DF30BFA%2F%29
onenote:https://microsoft.sharepoint.com/teams/IotHub22/SiteAssets/Iot%20Hub%20Notebook/PnP.one#Common%20swagger%20descriptions&section-id={FF56CB75-EE56-4A6F-AE61-85C3E2CE07A5}&page-id={9D7DB953-F12B-4BD6-B7EC-59619DF30BFA}&end)

Reply:
There is no "create" for twins, since the creation happens implicitly as soon as a device identity is created on the hub. So the above statement is not really relevant here. Let me know if you feel otherwise.

Reply:
This answers my previous question also then :) Is it worth mentioning in the PUT above?

Reply:
Do you still have a question here @vinagesh ?

Reply:
No